worlds/hk/Options.py

```diff
@@ -203,19 +203,27 @@ class MaximumCharmPrice(MinimumCharmPrice):
 
 
 class RandomCharmCosts(Range):
-    """Total Notch Cost of all Charms together. Set to -1 for vanilla costs. Vanilla sums to 90.
-    This value is distributed among all charms in a random fashion."""
+    """Total Notch Cost of all Charms together. Vanilla sums to 90.
+    This value is distributed among all charms in a random fashion.
+    Special Cases:
+    Set to -1 for vanilla costs.
+    Set to -2 to shuffle around the vanilla costs to different charms."""
 
     display_name = "Randomize Charm Notch Costs"
-    range_start = -1
+    range_start = -2
     range_end = 240
     default = -1
     vanilla_costs: typing.List[int] = vanilla_costs
     charm_count: int = len(vanilla_costs)
 
     def get_costs(self, random_source) -> typing.List[int]:
```

Comment:
random_source should probably be typed

Reply:
Ehh maybe seperating these into two options to make it more clear without having to read comments/descriptions.
`random charm notch costs` could be an option with `vanilla`, `vanilla_shuffled`, `range`
And then `charm_notch_range` could be the current range variable. Without the negative values.

Reply:
Then we get into options that do nothing unless another one has a certain value, which personally I like even less.